src/pal/inc/unixasmmacrosarm.inc

```diff
@@ -147,11 +147,15 @@ C_FUNC(\Name\()_End):
         __PWTB_StackAlloc = __PWTB_TransitionBlock
 
         .ifnc \pushArgRegs, DoNotPushArgRegs
-                PUSH_ARGUMENT_REGISTERS
+                push {r0 - r3}
+                .save {r0 - r3}
```

Comment:
The stack unwind directives should be part of the `PUSH_ARGUMENT_REGISTERS` macro - similar to how it is done on amd64.